README.md

```diff
@@ -700,6 +702,44 @@ In this example, all saves to `MyTable` will raise an `Dynamoid::Errors::StaleOb
 
 Calls to `update` and `update!` also increment the `lock_version`, however they do not check the existing value. This guarantees that a update operation will raise an exception in a concurrent save operation, however a save operation will never cause an update to fail. Thus, `update` is useful & safe only for doing atomic operations (e.g. increment a value, add/remove from a set, etc), but should not be used in a read-modify-write pattern.
 
+
+### Backoff strategies
+
+
+You can use several methods what do efficiently in batch mode like `.find_all` and `.import`.
+
+By some reasons, for instanse table throughput limit is exceeded, some items could be not processed.
+By default operations will be re-run to process this items.
+
+Recommended way to handle throughput limits exceeding and throttling on the table is to use exponential backoff.
+
+There are two default backoff strategies - constant delay and truncated binary exponential backoff.
```

Comment:
Default is a loaded term, let's use built-in instead, since the default is actually `nil`.

> There are two default built-in strategies - constant delay and truncated binary exponential backoff.